src/picknik_ur_gazebo_config/objectives/tree_nodes_model.xml

```diff
@@ -0,0 +1,25 @@
+<root>
```

Comment:
Should these be added to the core repo's `tree_nodes_model.xml` instead of making a new file here?

Reply:
This file is for the [sub-tree](https://github.com/PickNikRobotics/moveit_studio_ws/blob/c372d810ca4e8e439c9d28cfcd03960d7dbdfcb6/src/picknik_ur_gazebo_config/objectives/teach_apriltag_grasp_offset.xml) that only exists in this site_config.